emission/export/export.py

```diff
@@ -0,0 +1,53 @@
+import logging
+logging.basicConfig(level=logging.DEBUG)
+import gzip
+
+import uuid
+import json
+import bson.json_util as bju
+import emission.storage.timeseries.timequery as estt
+import emission.storage.timeseries.abstract_timeseries as esta
+import emission.storage.timeseries.cache_series as estcs
+import emission.net.usercache.abstract_usercache as enua
+def export(user_id, ts, start_ts, end_ts, file_name, ma_bool):
+        if ma_bool:
+                ma_time_query = estt.TimeQuery("metadata.write_ts", start_ts, end_ts)
+                uc = enua.UserCache.getUserCache(user_id)
+                ma_entry_list = uc.getMessage(["background/motion_activity"], ma_time_query)
+        else: 
+                ma_entry_list = [] 
+        loc_time_query = estt.TimeQuery("data.ts", start_ts, end_ts)
+        loc_entry_list = list(estcs.find_entries(user_id, key_list=None, time_query=loc_time_query))
+        trip_time_query = estt.TimeQuery("data.start_ts", start_ts, end_ts)
```

Comment:
can you please fix the indentation here as we discussed?
The convention is 4 spaces. This looks like it is more.
Leaving it like this will cause other editors to "help" by fixing it, which will again mess up the source control history.